products/datagrid/overview.adoc

```diff
@@ -1,4 +1,5 @@
 :awestruct-layout: product-overview
+:awestruct-ignore_export: true
```

Comment:
I added them to all of the product overview pages. Some of the overview pages had videos on them, but since we now don't render them from anymore, we don't really need to export them.